### DISCUSSION TITLE
CoPilot Unable to be Used in IDE (VS Studios) Your GitHub Copilot setup is incomplete.

### DISCUSSION THREAD

**Discussion by DrSkywalker** [community/community#77331] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Errors: Your GitHub Copilot setup is incomplete. Select your preferences below to start using GitHub Copilot.

GitHub Copilot 1.137.0.0 (v1.137.0.0@2f127d06d)
GitHub Copilot Agent 1.137.0 (1.137.0)
[DEBUG] [agent] [2023-11-29T16:47:54.673Z] Agent service starting
[DEBUG] [agent] [2023-11-29T16:47:54.885Z] Telemetry initialized
[INFO] [auth] [2023-11-29T16:47:55.792Z] Failed to get copilot token due to 401 status. Please sign out and try again.
Auth Status: NotSignedIn
[INFO] [auth] [2023-11-29T16:47:55.892Z] Failed to get copilot token due to 401 status. Please sign out and try again.
[INFO] [auth] [2023-11-29T16:47:57.521Z] Invalid copilot token: missing token: 403 
[INFO] [auth] [2023-11-29T16:47:57.664Z] Invalid copilot token: missing token: 403 
Auth Status: NotAuthorized
You don't have access to GitHub Copilot.

I Have the Student Developer Pack and Access to Github Copilot.

Wondering if Anyone is having issues 11/29/2023

Also I have Revoked the token
Reinstall extension

So im at a loss any help??

**Comment by Akash1134** [comment]
Hi @DrVarsity You might need to reverify your access to both the Student developer pack and Copilot. The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by OshinoShinobu-Chan** [comment, marked as answer]
You can select the choice and retry, this works on me.
![image](https://github.com/community/community/assets/128981786/99772fa2-ed44-4ff8-88cd-5491d984298f)

**Reply by samiri123** [reply]
that worked for me too. thx

**Reply by Majdar133ouk** [reply]
thanks

**Reply by gecuasapaz** [reply]
tks! its work for me

**Reply by GabryelKadmo** [reply]
ty still works

**Reply by Kajivan** [reply]
Thank you so much it finally worked

**Reply by WZHLEAVES** [reply]
Thank you so much

**Reply by RebornXD** [reply]
ty it works for me!

**Reply by shanks-20million** [reply]
it did, but not sure why, though!!

**Reply by DanJSal** [reply]
I spent hours troubleshooting and this was the only thing that worked. Thanks!

**Reply by vasylkomarnytskyi** [reply]
Thank you so much! It work for me.

**Reply by roshan0099** [reply]
workss, thanks a ton

**Reply by Thomas-Gustafsson** [reply]
This still works
